build.sbt

```diff
@@ -22,14 +22,15 @@ lazy val root = (project in file(".")).enablePlugins(
 )
 
 
-scalaVersion := "2.11.7"
+scalaVersion := "2.11.8"
+scalacOptions ++= Seq("-feature")
```

Comment:
This is set on nearly all our other apps. http://stackoverflow.com/questions/15533706/how-to-get-more-information-about-feature-flag-warning